bin/templates/templates.xml

```diff
@@ -251,4 +251,26 @@
             </ul>
         ]]></description>
     </template>
+    <template isTestPlan="true">
+        <name>Simple HTTP request</name>
+        <fileName>/bin/templates/simple-http-request-test-plan.jmx</fileName>
+        <description><![CDATA[
+        <h1>Template for simple HTTP request</h1>
+        <h2>JMeter Configuration</h2>
+        <p>Configure your HTTP request, set:
+            <ul>
+                <li>Full URL</li>
+                <li>Content Type</li>
+				<li>HTTP method</li>
```

Comment:
Nobody will see it, but could you try to keep the spacepolice happy?

Reply:
@FSchumacher I don't see an issue with spaves